packages/publisher/package.json

```diff
@@ -34,7 +34,6 @@
     "test": "../../node_modules/.bin/jest --config ../../jest.config.js packages/publisher",
     "clean": "node -r source-map-support/register dist/clean.js",
     "get-definitely-typed": "node -r source-map-support/register dist/get-definitely-typed.js",
-    "parse": "node -r source-map-support/register dist/parse-definitions.js",
```

Comment:
Unrelated and not for this PR, but we should really drop `source-map-support` everywhere; `node --enable-source-maps`  has been around since Node 12, stable since node 14.18.

Reply:
Sent #767 since it took 2 seconds.